opencraft/settings.py

```diff
@@ -430,7 +430,7 @@
 STABLE_CONFIGURATION_REPO_URL = env(
     'STABLE_CONFIGURATION_REPO_URL', default=DEFAULT_CONFIGURATION_REPO_URL
 )
-STABLE_CONFIGURATION_VERSION = env('STABLE_CONFIGURATION_VERSION', default='opencraft-release/ironwood.2')
+STABLE_CONFIGURATION_VERSION = env('STABLE_CONFIGURATION_VERSION', default='opencraft-release/juniper.2')
```

Comment:
@demid, `DEFAULT_OPENEDX_RELEASE` has to be updated as well. We also need to create a PR against the ansible secrets repository where the configuration to set these environment variables are stored.